core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -252,6 +252,8 @@
       Name.USER_FILE_WORKER_CLIENT_POOL_GC_THRESHOLD_MS, 300 * Constants.SECOND_MS),
   USER_FILE_WRITE_LOCATION_POLICY(Name.USER_FILE_WRITE_LOCATION_POLICY,
       "alluxio.client.file.policy.LocalFirstPolicy"),
+  USER_FILE_WRITE_CAPACITY_RESERVED_SIZE_BYTES(
```

Comment:
This property only modifies the new policy, so what do you think of naming it
`USER_FILE_WRITE_AVOID_EVICTION_POLICY_RESERVED_BYTES`?